src/SFA.DAS.FAA.Web/Views/Home/GetHelp.cshtml

```diff
@@ -0,0 +1,38 @@
+﻿@{
+    Layout = "_Layout";
+    ViewBag.Title = "Get help with Find an apprenticeship – Find an apprenticeship – GOV.UK";
+}
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l govuk-!-margin-bottom-8">
+            Get help with Find an apprenticeship
+        </h1>
+
+        <h2 class="govuk-heading-m">
+            Contact the Apprenticeship service
+        </h2>
+        <p class="govuk-body">
+           Contact us if you're having a problem with the service.
+        </p>
+        
+        <h2 class="govuk-heading-s">
+            Call us
+        </h2>
+        <p class="govuk-body">
+            Call 0800 015 0400
+            <br/>
+            Open Monday to Friday, 9am to 5pm
+            <br/>
+            Free from UK mobiles and landlines
+        </p>
+
+        <h2 class="govuk-heading-s">
+            Email us
+        </h2>
+        <p class="govuk-body">
```

Comment:
Might have missed this last time, but is there a reason for the double <p class="govuk-body"> around the link rather than a single?

Reply:
Ok, github stripped the html I typed! It typed in the paragraph with govuk-body.

Reply:
remove extra p tag